Exiled.API/Features/Room.cs

```diff
@@ -99,6 +99,35 @@ public Color Color
         /// <param name="duration">Duration in seconds.</param>
         public void TurnOffLights(float duration) => FlickerableLightController?.ServerFlickerLights(duration);
 
+        /// <summary>
+        /// Locks all the doors in the room.
+        /// </summary>
+        /// <param name="duration">Duration in seconds.</param>
+        /// <param name="locktype">DoorLockType of the lockdown.</param>
+        public static void LockDown(float duration, DoorLockType locktype = DoorLockType.Regular079)
```

Comment:
static methods should be above non-static ones.

Also this should be renamed to LockAll() and the documentation changed to indicate that it affects all rooms.